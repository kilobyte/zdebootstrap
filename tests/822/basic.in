oneline: foo bar
multiline1: line0
 line1
 line2
multiline2:
 line1
 line2
