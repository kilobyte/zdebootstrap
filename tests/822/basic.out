multiline1: line0
 line1
 line2
multiline2:
 line1
 line2
oneline: foo bar
